Analysis/Tasks/PWGHF/qaTaskEfficiency.cxx

```diff
@@ -65,84 +61,74 @@ void makelogaxis(T h)
 
 /// Task to QA the efficiency of a particular particle defined by particlePDG
 template <o2::track::pid_constants::ID particle>
-struct QATrackingEfficiencyPt {
+struct qaTrackingEfficiency {
```

Comment:
As I mentioned in https://github.com/AliceO2Group/AliceO2/pull/5556,
[Capitalization Rules](https://rawgit.com/AliceO2Group/CodingGuidelines/master/naming_formatting.html?showone=General_Naming_Rules#General_Naming_Rules):
> Variables and functions start with a lowercase letter.
Everything else in C++ code (type names, constant expressions) starts with an uppercase letter.